infos/anfi-faq.md

```diff
@@ -13,13 +13,13 @@ nachzuschauen und bei den
 [Studienberatern](https://uni-tuebingen.de/de/74360)
 nachfragen. Wer Fragen hat, die hier nicht beantwortet werden, darf sie
 uns gerne stellen: Kontaktmöglichkeiten stehen auf unserer
-[Kontaktseite](/kontakt/)
+[Kontaktseite](/kontakt/) Seite
```

Comment:
This will evaluate to 
Kontaktseite Seite
where Kontaktseite is a hyperlink. Does this make any sense?

Reply:
Actually it doesn't. Either leave it at `[Kontaktseite](/kontakt/)` or make it something completely different like "[…] darf sie uns gerne auf der Unterseite `[Kontakt](/kontakt/)` stellen. This way it's just confusing and the reader stumbles over the repetition.